js/demo-apps/packages/react-front-end/src/common/components/Header.tsx

```diff
@@ -73,27 +114,40 @@ export const Header: React.FunctionComponent<{
   return (
     <AppBar position="fixed" className={classes.appBar}>
       <Toolbar>
-        <Typography variant="h6" noWrap className={classes.title}>
-          LunaSec Example App
-        </Typography>
-        <ToggleButtonGroup
-          className={classes.toggleButtonGroup}
-          exclusive
+        {/* hamburger icon shows the drawer on click */}
+        {!onDesktop ? (
```

Comment:
nit: I prefer the syntax of something like this. It's flatter and easier to follow.
```tsx
function render() {
  const isDesktopMode = useMediaQuery(theme.breakpoints.up('md'));

  const mobileNavContent = (
    <IconButton
      className={classes.sideBarButton}
      edge="start"
      aria-label="open drawer"
      onClick={() => setSidebarOpen(!sidebarOpen)}
      size="large"
    >
      <MenuIcon />
    </IconButton>
  );
  
  const desktopNavContent = (
    <Typography variant="h6" noWrap className={classes.title}>
      LunaSec Example App
    </Typography>
  );

  const navContent = isDesktopMode ? desktopNavContent : mobileNavContent;
  
  return (
    <AppBar position="fixed" className={classes.appBar}>
      <Toolbar>
        {navContent}
        /* ... */
      </Toolbar>
    </AppBar>
  );
}
```

Alternatively, another style would be to just extract it into a functional component.

```tsx
function NavBar() {
  
  const isDesktopMode = useMediaQuery(theme.breakpoints.up('md'));
  
  if (isDesktopMode === false) {
    return (
      <IconButton
        className={classes.sideBarButton}
        edge="start"
        aria-label="open drawer"
        onClick={() => setSidebarOpen(!sidebarOpen)}
        size="large"
      >
        <MenuIcon />
      </IconButton>
    );
  }
  
  return (
    <Typography variant="h6" noWrap className={classes.title}>
      LunaSec Example App
    </Typography>
  );
}
```